### DISCUSSION TITLE
Welcome to meghna-hugo Discussions!

### DISCUSSION THREAD

**Discussion by somratpro** [themefisher/meghna-hugo#146] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wondercomm** [comment]
Hello

I have a question: How do you update the favicon for the theme? I replaced the favicon.png in the static/images directory. It is showing the correct favicon image when I click it in github but it hasn't taken effect on my website. What is the correct way of updating the favicon?

Btw, Brilliant Theme, love it!

**Comment by divinerites** [comment]
I guess you have to clear your browser cache


>I replaced the favicon.png in the static/images directory. It is showing the correct favicon image when I click it in github but it hasn't taken effect on my website.

**Comment by wondercomm** [comment]
How do I increase the font size of the menu items? (about, services, contact, etc.) ?
